registry/quilt_server/views.py

```diff
@@ -347,6 +349,13 @@ def wrapper(*args, **kwargs):
                         raise ApiException(requests.codes.server_error, "Server error")
                 except (ConnectionError, requests.RequestException) as ex:
                     raise ApiException(requests.codes.server_error, "Server error")
+
+            if require_admin and (not g.auth or not g.auth.is_admin):
```

Comment:
Don't need to check for `not g.auth` - it always gets set at the top of the function